website/docs/r/express_route_circuit_peering.html.markdown

```diff
@@ -104,14 +154,18 @@ A `microsoft_peering_config` block contains:
 
 A `ipv6` block contains:
 
-* `microsoft_peering` - (Required) A `microsoft_peering` block as defined below.  
-
 * `primary_peer_address_prefix` - (Required) A subnet for the primary link.
 
 * `secondary_peer_address_prefix` - (Required) A subnet for the secondary link.
+ 
+* `ipv6_enabled` - (Optional) A boolean value indicating whether the IPv6 peering is enabled. Defaults to `true`.
+
+* `microsoft_peering` - (Optional) A `microsoft_peering` block as defined below.  
 
 * `route_filter_id` - (Optional) The ID of the Route Filter. Only available when `peering_type` is set to `MicrosoftPeering`.
 
+~> **NOTE:** `ipv6` can be specified when `peering_type` is `MicrosoftPeering` or `AzurePrivatePeering`
```

Comment:
we should validate tyhis?

Reply:
this is validated : https://github.com/hashicorp/terraform-provider-azurerm/pull/17338/files#diff-f299371fb1652bb35b191f998f475a02d45eb4bea3a3f2357d32766a9ebb948fR301